src/common/fam_libfabric.cpp

```diff
@@ -708,32 +704,21 @@ int fabric_completion_wait_multictx(Fam_Context *famCtx, fi_context *ctx,
             FI_CALL(ret, fi_cq_readerr, famCtx->get_txcq(), &err, 0);
 
             if (ret == 1) {
-                for (int j = 0; j < count; j++) {
```

Comment:
fabric_completion_wait_multictx - This function is not being used any more. 
You can keep this function unchanged or keep it commented for any future use.

Reply:
Currently this function has been commented.